be1-go/popcha/storage/mod.go

```diff
@@ -0,0 +1,73 @@
+package storage
+
+import (
+	"github.com/rs/zerolog"
+	"golang.org/x/xerrors"
+	"sync"
+)
+
+/*
+Storage defines the in-memory store of the authorization server. It is used for
+nonce replay prevention, Pop-Token-To-Identifier uniqueness as well as for keeping
+track of ongoing requests.
+*/
+type Storage struct {
+	lock                 *sync.Mutex
+	popTokenToIdentifier map[string]string //hash map enforcing uniqueness on (PopToken,identifier)
+	nonces               map[string]struct{}
+	log                  zerolog.Logger
+}
+
+// NewStorage initializes a Storage object given a logger
+func NewStorage(log zerolog.Logger) (*Storage, error) {
+	return &Storage{
+		lock:                 &sync.Mutex{},
+		popTokenToIdentifier: make(map[string]string, 0),
```

Comment:
Is there a reason why you specify the 0 argument in `make()` for some usages and others not ? I think 0 is already a default value but I'm not 100% sure about that.